main.go

```diff
@@ -166,6 +218,24 @@ func main() {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+
+	// check if cred request API exists in the cluster before creating a cred request
+	c := mgr.GetClient()
+	credReqCRDExists, err := DoesCRDExist("CredentialsRequest")
+	if err != nil {
+		setupLog.Error(err, "problem checking the existence of CredentialRequests CRD")
+		os.Exit(1)
+	}
+
+	if credReqCRDExists {
+		// create cred request
+		if err := c.Create(context.TODO(), credReq); err != nil {
+			if !errors.IsAlreadyExists(err) {
+				setupLog.Error(err, "unable to create credRequest")
```

Comment:
This might be more reliable?
```
❯ oc describe co cloud-credential
Name:         cloud-credential
Namespace:    
Labels:       <none>
Annotations:  exclude.release.openshift.io/internal-openshift-hosted: true
              include.release.openshift.io/self-managed-high-availability: true
API Version:  config.openshift.io/v1
Kind:         ClusterOperator
Spec:
```